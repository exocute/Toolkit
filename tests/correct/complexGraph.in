Graph com.zink.Imager

	Activity QT com.zinke.Imager.QuadTile
	Import com.zink.ImageBuffer 
	Export List[com.zink.ImageBuffer]

	Activity SF com.zink.exocute.SpliterFlattener
	Import List[com.zink.ImageBuffer]
	Export com.zink.ImageBuffer

	Activity FO com.zink.Foci
	Import com.zink.ImageBuffer
	Export com.zink.ClusterCentre

	Activity RGBM com.zink.com.zink.RGBMusher
	Import com.zink.ImageBuffer
	Export com.zink.RGB

	Activity FC com.zink.FociCombiner
	Import com.zink.RGB
	Import com.zink.ClusterCentre
	Export com.zink.WeightedCluster

	connection QT->SF:SF->FO,RGBM:RGBM->FC:FO->FC