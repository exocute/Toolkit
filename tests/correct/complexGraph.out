Name: com.zink.Imager
Id: SF, Name: com.zink.exocute.SpliterFlattener, Import: List[com.zink.ImageBuffer], Export: com.zink.ImageBuffer
Id: QT, Name: com.zinke.Imager.QuadTile, Import: com.zink.ImageBuffer, Export: List[com.zink.ImageBuffer]
Id: RGBM, Name: com.zink.com.zink.RGBMusher, Import: com.zink.ImageBuffer, Export: com.zink.RGB
Id: FC, Name: com.zink.FociCombiner, Import: com.zink.ClusterCentre,com.zink.RGB, Export: com.zink.WeightedCluster
Id: FO, Name: com.zink.Foci, Import: com.zink.ImageBuffer, Export: com.zink.ClusterCentre
SF:RGBM,FO
QT:SF
RGBM:FC
FO:FC